repo2docker/buildpacks/base.py

```diff
@@ -414,12 +415,18 @@ def get_start_script(self):
         """
         return None
 
+    @property
+    def binder_dir(self):
+        if self._binder_dir is None:
+            for dirname in ['binder', '.binder']:
+                if os.path.exists(dirname):
```

Comment:
I'd switch to `os.path.isdir` to make it explicit that we need `dirname` to be a directory and not any kind of path. I think with this we get a better error message in the exception "for free". 

https://docs.python.org/3/library/os.path.html#os.path.isdir